man/systemd-oomd.service.xml

```diff
@@ -101,6 +102,24 @@
     something like <filename>user@$UID.service</filename> may prefer a much lower value like 40%.</para>
   </refsect1>
 
+  <refsect1>
+    <title>Options</title>
+
+    <variablelist>
+      <varlistentry>
+        <term><option>--dry-run</option></term>
+
+        <listitem><para>Do a dry run of <command>systemd-oomd</command>: when a kill is triggered, print it
+        to the log instead of killing the cgroup.</para></listitem>
+      </varlistentry>
+    </variablelist>
+
+    <xi:include href="standard-options.xml" xpointer="help" />
+    <xi:include href="standard-options.xml" xpointer="version" />
+  </refsect1>
+
+  <xi:include href="common-variables.xml" />
```

Comment:
I don't think this is necessary...

Reply:
I went looking for it to figure out how to set it to debug log level, so it would have been useful to me earlier today. I can remove it though.

Reply:
At least, please include entries only really useful for oomd.
But, your another PR was already merged. Hence, I think we can drop this line entirely.